app/mailers/order_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class OrderMailer < ApplicationMailer
+  default from: 'valar@employeerp.co.uk'
```

Comment:
this is not needed, as you have the same line in `ApplicationMailer`

Reply:
done it's gone